schema/data/prod/json_schema/contact.json

```diff
@@ -2,7 +2,7 @@
   "schema": {
     "$schema": "http://json-schema.org/draft-07/schema",
     "type": "object",
-    "required": ["givenName", "familyName", "email", "phone"],
+    "required": ["givenName", "familyName", "email"],
     "properties": {
       "givenName": {
         "type": "string",
```

Comment:
Why was 'phone' removed as a requirement here?

Reply:
Currently, the `contactSchema.ts` is being used as the schema for the contact form instead of `contact.json`. To ensure consistency, I made them identical. Upon reviewing the PROD data, I noticed multiple null values for the phone field. I believe that it was not necessary from the start. However, please correct me if my assumption is incorrect.